src/coreimage.cs

```diff
@@ -430,7 +430,7 @@ interface CIContext_ImageRepresentation {
 		[Wrap ("GetJpegRepresentation (This, image, colorSpace, options.GetDictionary ()!)")]
 		[return: NullAllowed]
 		NSData GetJpegRepresentation (CIImage image, CGColorSpace colorSpace, CIImageRepresentationOptions options);
-
+ 
```

Comment:
Not sure if this is an extra space with this newline or if it is just a newline!